pkg/util/ovfcache/ovfcache.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package ovfcache
+
+import (
+	"context"
+	"time"
+
+	"github.com/vmware/govmomi/ovf"
+
+	"github.com/vmware-tanzu/vm-operator/pkg/util/ovfcache/internal"
+)
+
+// VersionedOVFEnvelope is an OVF envelope with an associated content version.
+type VersionedOVFEnvelope = internal.VersionedOVFEnvelope
+
+// GetterFn returns an OVF envelope for a provided library item ID.
+type GetterFn = internal.GetterFn
```

Comment:
I don't see these types being referenced by any external callers. Is there any reason to declare them here instead of keeping them in the internal file?

Reply:
The getter function is as it is used in SetGetter.